views/js/controller/creator/helpers/qtiTest.js

```diff
@@ -82,13 +82,29 @@ define(['lodash', 'i18n'], function(_, __){
             };
         },
 
+        /**
+         * Gives you a validator that check QTI id format of the test (it is different from the others...)
+         * @returns {Object} the validator
+         */
+        testidFormatValidator : function testidFormatValidator(){
+            var qtiTestIdPattern = /^\S+$/;
```

Comment:
So this is is a valid identifier `Test-1ùµd*$}` ? This however is valid in the form, but an InvalidArgumentException is thrown by (vendor/qtism/qtism/qtism/data/AssessmentTest.php) at line 192 if I try to save the test. So something is wrong ...